roles/orangelight/tasks/main.yml

```diff
@@ -2,17 +2,17 @@
 # tasks file for roles/orangelight
 
 - name: orangelight | update rubygems
-  ansible.builtin.command: gem update --system 3.5.10
+  ansible.builtin.command: gem update --system 3.6.1
   become: true
   register: update_rubygems
   changed_when: '"Latest version already installed. Done." not in update_rubygems.stdout'
   tags: rubygems
 
 - name: orangelight | install default bundler
-  ansible.builtin.command: gem install --default bundler:2.5.10
+  ansible.builtin.command: gem install --default bundler:2.6.1
   become: true
   register: install_default_bundler
-  changed_when: '"Fetching bundler-2.5.10.gem" in install_default_bundler.stdout'
+  changed_when: '"Fetching bundler-2.6.1.gem" in install_default_bundler.stdout'
   tags: rubygems
 
 - name: orangelight | remove old default bundler
```

Comment:
@sandbergja I think we need to remove the default bundler 2.5.10

Reply:
Ah!  Thanks for catching that.  I force pushed.